medicines/web/copy/about.md

```diff
@@ -45,8 +45,8 @@ by either the European Medicines Agency (EMA) or the responsible
 authority of another EU member state. The PARs for these medicines can
 be found on the following websites:
 
-- [European Medicines Agency (EMA)][ema medicine]
-- [Head of Medicines Agency][head of med]
+[European Medicines Agency (EMA)][ema medicine]
+[Head of Medicines Agency][head of med]
```

Comment:
this won't display them properly - they'll be on the same line

![Screenshot 2019-12-12 at 17 18 46](https://user-images.githubusercontent.com/7150842/70734078-7697f900-1d03-11ea-91d8-a6b11a2d9630.png)